public/update_team.php

```diff
@@ -0,0 +1,152 @@
+<?php require_once(dirname(__FILE__)."/../modules/ensure_logged_in.php"); ?>
+<?php include "templates/header.php"; ?>
+
+<?php 
+$lecture_id = $_GET['id'];
+
+if(get_current_role() == 'instructor') {
+
+require_once "../modules/models/user.php";
+require_once "../modules/models/section.php";
+require_once "../modules/models/section_student.php";
+require_once "../common.php";
+
+try {
+    $teams = Team::includes('team_member')->where(array('lecture_id' => $lecture_id));
+
+} catch (PDOException $error) {
+    echo "<br>" . $error->getMessage();
+}
+
+try {
+    $course_sections = Section::includes('section_students')->where(array('lecture_id' => $lecture_id));
+
+} catch (PDOException $error) {
+    echo "<br>" . $error->getMessage();
+}
+
+
+$students_in_teams = array();
+
+foreach($teams as $t) {
+    foreach ($t->team_member as $member ) {
+        array_push($students_in_teams, $member->user_id);
+    }
+}
+
+$course_students = array();
+
+foreach ($course_sections as $section) {
+    try {
+        $course_section_student = $section->section_students;
+        foreach ($course_section_student as $student){
+            array_push($course_students, $student->user);
+        }
+        
+    } catch (PDOException $error) {
+        echo "<br>" . $error->getMessage();
+    }
+}
+
+$available_students = array();
+foreach ($course_students as $student) {
+    if (!in_array($student->id, $students_in_teams)){
+        array_push($available_students, $student);
+    }
+    
+}
+
+if (isset($_POST["submit"])) {
+
+    $team_to_edit_id = $_POST['team'];
+    if (isset($_POST['team_member_delete'])){
+        $students_to_remove = $_POST['team_member_delete'];
+    }
+    else {
+        $students_to_remove = null;
+    }
+    
```

Comment:
`$students_to_remove = $_POST['team_member_delete'] ?? null;` should work instead of the entire if else statement

Reply:
or maybe even $students_to_remove = $_POST['team_member_delete'] ?? [];

this way you can skip the null check below